onnx/defs/reduction/defs.cc

```diff
@@ -73,7 +73,7 @@ The type of the output tensor is integer.)DOC";
     schema.SetDoc(doc);
     schema.Attr(
         "axis",
-        "The axis in which to compute the arg indices",
+        "The axis in which to compute the arg indices. By default it shall compute along the last axis.",
```

Comment:
There's already a default "0" specified. Are you changing the spec to use last axis anyway? if yes, you'll need to remove the default value specified below.

Reply:
Let's keep 0 as default value, unless we have good reason for that.

Reply:
resolved